documentation/modules/appendix_crds.adoc

```diff
@@ -1033,7 +1033,7 @@ It must have the value `jmxPrometheusExporter` for the type `JmxPrometheusExport
 |Must be `jmxPrometheusExporter`.
 |valueFrom
 |xref:type-ExternalConfigurationReference-{context}[`ExternalConfigurationReference`]
-|ConfigMap entry where the Prometheus JMX Exporter configuration is stored. 
+|ConfigMap entry where the JMX Prometheus Exporter configuration is stored.
```

Comment:
I think `Prometheus JMX Exporter` is the proper name. It's called JMX Exporter and it's from the Prometheus project.

Reply:
I see in your last commit you used `JMX Prometheus Exporter`. Sorry if my message was confusing but I meant we should use `Prometheus JMX Exporter` instead.

Reply:
@fvaleri I believe this is still not addressed.

Reply:
I'll fix that. It is so easy to invert them as the metricsConfig type is and spec class name are prefixed with JMXPrometheusExporter.